phpBB/phpbb/mention/helper/mention_helper.php

```diff
@@ -0,0 +1,207 @@
+<?php
+/**
+ *
+ * phpBB mentions. A feature for the phpBB Forum Software package.
+ *
+ * @copyright (c) 2016, phpBB, https://www.phpbbextensions.io
```

Comment:
Invalid copyright, please change to:
`@copyright (c) phpBB Limited <https://www.phpbb.com>`